src/Sulu/Bundle/WebsiteBundle/Command/DumpSitemapCommand.php

```diff
@@ -120,16 +135,33 @@ private function dumpSitemap(PortalInformation $portalInformation)
             $portalInformation->setUrl(str_replace('{host}', $this->defaultHost, $portalInformation->getUrl()));
         }
 
-        if (!$this->sitemapProviderPool->needsIndex()) {
+        $filePath = $this->sitemapRenderer->getIndexDumpPath(
+            $this->scheme,
+            $portalInformation->getWebspaceKey(),
+            $portalInformation->getLocale(),
+            $portalInformation->getHost()
```

Comment:
Maybe pass the `PortalInformation`? Would maybe generally be a good idea, because the sitemap is generated for a `PortalInformation`.